Dockerfile.arm64

```diff
@@ -0,0 +1,37 @@
+# Build stage
+FROM golang:1.9.7 as build
```

Comment:
the other `Dockerfile` is using `golang:1.10.4` and `alpine:3.8`, can we use those here too?

Reply:
Sure, I'll give it a go :+1:

Reply:
Applied your suggestion, thanks! :+1: